docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,79 +2,39 @@
 
 These workflows for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
-### Preferred Instructions
+_Last updated 27-Mar-2024_
 
-_Added 2020-05-06_
-
-1. Open the `src/ontology/imports/*_terms.txt`
-2. Add term to this file
+1. Open the `src/ontology/imports/manual_seed.txt`
```

Comment:
In my mind, someone following these instructions should know they should fetch from master, create a branch, and then push to a branch. 
I know we have these basic steps in some other instructions, and therefore we are inconsistent in adding these or not. 
I don't know that it is needed, but I can be convinced otherwise. 
@nicolevasilevsky, do you have an opinion on this?

Reply:
I personally like it when instructions are really explicit like this, especially when I am new to something (or for other new curators), so I am in favor of including it